app/Providers/AppServiceProvider.php

```diff
@@ -24,14 +23,6 @@ public function boot()
      */
     public function register()
     {
-        // @TODO: This should be registered in Gateway's service provider!
```

Comment:
Is this registered in Gateway's service provider now?

Reply:
[Yep](https://github.com/DoSomething/gateway/blob/d1479758bb276e8ddb950d49cc3827322684a819/src/Laravel/GatewayServiceProvider.php#L46-L48)!